src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:20811/organizations/default-organization/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:20811/project/issues?id=sample&branch=feature4&issues=AWDhhps8rUevNsX_PmM7&open=AWDhhps8rUevNsX_PmM7)

[ISSUE_KEY]: AWDhhps8rUevNsX_PmM7